src/base/ApplicationECSService.ts

```diff
@@ -437,4 +465,96 @@ export class ApplicationECSService extends Resource {
       tags: { ...this.config.tags, type: name },
     });
   }
+
+  private createEfsMount(efsFs: EFSProps) {
+    const ingress: vpc.SecurityGroupIngress[] = [
+      {
+        fromPort: 2049,
```

Comment:
Just curious as to why the port numbers are hard coded? Will these always remain the same?

Reply:
Added comment - port 2049 is unchangable